### PR TITLE
chore: Edit APIGatewayV2Request per "LAMBDA-PROXY" event

### PR DESCRIPTION
**Pull request by idelfonsog2**
Edit a couple of the properties in the APIGatewayV2Request

### Motivation:

I was getting Decoding errors when invoking my lambda using APIGateway Event Request. I looked at the [AWS Documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format) and [Serverless documentation](https://www.serverless.com/framework/docs/providers/aws/events/apigateway#example-lambda-proxy-event-default)  and notice a different object from what is currently in the package

### Modifications:

Edit a couple of the properties in the APIGatewayV2Request

### Result:

My lambda is able to receive the Event from APIGateway along with the body I included in my http request

### CODE REVIEW

**Comment by 0xTim** [comment]
This is a pretty breaking change that would break any `APIGatewayV2Request` that doesn't use a proxy. We'll probably need a new Request type

**Comment by idelfonsog2** [comment]
I was thinking that was going to be the case given that I did not find any proxy related piece of info about it in the package 👍. I will add it separately and update the PR

File: Sources/AWSLambdaEvents/APIGateway+V2.swift [hunk 1]
```diff
@@ -23,54 +23,122 @@ public struct APIGatewayV2Request: Codable {
...
+        
+        case context = "requestContext"
+        case stageVariables
+        
+        case body
+        case isBase64Encoded
+    }
+}
+
+public struct APIGatewayV2ProxyRequest: Codable {
```
Comment (fabianfett) [inline Sources/AWSLambdaEvents/APIGateway+V2.swift:88]: I think this should go into a separate file and be called `LambdaProxyEvent`.

File: Sources/AWSLambdaEvents/APIGateway+V2.swift [hunk 2]
```diff
@@ -23,54 +23,122 @@ public struct APIGatewayV2Request: Codable {
...
+        case cookies
+        case headers
+        case queryStringParameters
+        case pathParameters
+        
+        case context = "requestContext"
+        case stageVariables
+        
+        case body
+        case isBase64Encoded
```
Comment (fabianfett) [inline Sources/AWSLambdaEvents/APIGateway+V2.swift:84]: We shouldn't do this here. As stated below, create a completely separate file and type.

File: Sources/AWSLambdaEvents/APIGateway+V2.swift (hunk 1 above)
Comment (idelfonsog2) [reply Sources/AWSLambdaEvents/APIGateway+V2.swift:88]: Will do

**Comment by idelfonsog2** [comment]
I opened a new PR just wanted to clean the commit history @fabianfett

**Pull request closed without merging**